File: lista03/ex03.c
/*
Faça um programa que receba como entrada um número inteiro n e exiba o resultado da
seguinte soma:

S = 1 + 1/2 + 1/3 + 1/4 + 1/n
*/

#include <stdio.h>

int main(){
    int i, n; 
    float soma = 0;

    printf("Digite um numero inteiro: ");
    scanf("%d",&n);

    for (i = 1; i <= n; i++){
        soma += (1/(float)i);
    }
    
    printf("\nValor da soma: %.3f",soma);

    return 0;
}

File: laboratorios/lab5/ex03.py
"""
Escreva um programa em que, dado um número N como entrada pelo teclado, 
exiba como saída todos os números pares na
sequência de 1 a N (Intervalo fechado).
"""

n = int(input('Digite um valor: '))

for i in range(1,n+1):
    if i % 2 == 0:
        print(i)